ee/clickhouse/queries/funnels/funnel.py

```diff
@@ -109,7 +120,28 @@ def _format_single_funnel(self, result, with_breakdown=False):
                         "breakdown_value": result[-1],
                     }
                 )
-                # important to not try and modify this value any how - as these are keys for fetching persons
+                # important to not try and modify this value any how - as these
+                # are keys for fetching persons
+
+                # Add in the breakdown to people urls as well
+                # TODO: handle cohorts?
```

Comment:
Is there something missing here with cohort handling? O.o

Reply:
possibly not, I saw that there was some specific handling of cohorts just above, but looking closer this is just for getting the cohort name for display in the frontend? 

I'll remove the comment

Reply:
Somehow I didn't push this change, will update in the frontend PR

Reply:
can be found [here](https://github.com/PostHog/posthog/pull/7099/commits/d0b7728f6e833363d7196d8bff9f661047d5a7af)